tests/de/all_personas.py

```
@@ -0,0 +1,32 @@
+import datetime
+
+from _gettsim_personas.persona_objects import Persona
+from gettsim_personas import (
+    einkommensteuer_sozialabgaben,
+    grundsicherung_für_erwerbsfähige,
+)
+
+ALL_PERSONA_SUBMODULES = [
+    einkommensteuer_sozialabgaben,
+    grundsicherung_für_erwerbsfähige,
+]
+
+
+def get_all_personas():
+    persona_objs = []
+    for submodule in ALL_PERSONA_SUBMODULES:
+        for name in dir(submodule):
+            obj = getattr(submodule, name)
+            if isinstance(obj, Persona):
+                persona_objs.append(obj)
+    return persona_objs
+
+
+def persona_year_pairs(start=2005, end=2025):
```

Can we make the end year dynamic in one way or another? Seems like this will be easy to forget, I'd rather see test failures if 2026 is not supported yet in GETTSIM than silently passing tests even though it does not work.

As a compromise, we could make it very salient (maybe here we could use a  release numbering with `<LATEST_SUPPORTED_YEAR>.<COUNTER>`)?

---

I think we should just do `END_YEAR = datetime.date.today().year`.